bin/millau/runtime/src/xcm_config.rs

```diff
@@ -159,6 +162,11 @@ pub type LocalOriginToLocation = (
 	SignedToAccountId32<RuntimeOrigin, AccountId, ThisNetwork>,
 );
 
+#[cfg(feature = "runtime-benchmarks")]
+parameter_types! {
+	pub ReachableDest: Option<MultiLocation> = Some(Parachain(1000).into());
```

Comment:
Sorry, I'm not familiar with this, but what is parachain 1000 ? Do we have a constant for it ?

Reply:
ok, we dont need that, so I removed that and put there just `todo!`,